src/suggestions.ts

```diff
@@ -1,132 +1,48 @@
-        title: 'Conta desconectada',
-        message: 'Realize novamente o seu login'
-    },
-    ENTITY_NOT_FOUND: {
-        type: 'ENTITY_NOT_FOUND',
-        title: 'Entidade não encontrada',
-        message: 'Tente novamente'
-    },
-    INVALID_INPUT: {
-        type: 'INVALID_INPUT',
-        title: 'Campo inválido',
-        message: 'Um ou mais campos inválidos'
-    },
-    INVALID_SUBSCRIPTION: {
-        type: 'INVALID_SUBSCRIPTION',
-        title: 'Plano não existente',
-        message: 'Esta conta não possui um plano vinculado, contate o administrador'
-    },
-    ENTITY_IS_STILL_REFERENCED: {
-        type: 'ENTITY_IS_STILL_REFERENCED',
-        title: 'Entidade referenciada',
-        message: 'Não é possivel realizar a operação, entidade ainda possui vínculos'
-    },
-    DUPLICATED_ENTITY: {
-        type: 'DUPLICATED_ENTITY',
-        title: 'Entidade duplicada',
-        message: [
-            'Um ou mais campos únicos informados',
-            'já estão cadastrados no sistema'
-        ].join(' ')
-    },
-    CANNOT_FINISH_REQUEST_WITH_ORDER: {
-        type: 'CANNOT_FINISH_REQUEST_WITH_ORDER',
-        title: 'O.S. vinculada pendente',
-        message: [
-            'Não é possível finalizar uma solicitação de',
-            'serviço se ela possui uma ordem de serviço aberta'
-        ].join(' ')
-    },
-    EMPLOYEE_INACTIVE_ON_CURRENT_BRANCH: {
-        type: 'EMPLOYEE_INACTIVE_ON_CURRENT_BRANCH',
-        title: 'Funcionário inativo',
-        message: 'Esse funcionário não está ativo nessa filial'
+    // When graphql finds a wrong field sent
+    if (message.includes('Unknown argument')) {
+        return {
+            type: 'SCHEMA_UNKNOWN_FIELD',
+            message: 'Parece que seu Keepfy pode estar desatualizado'
+        }
     }
 
+    return defaultKeepfyErrorTrusted
 }
 
-export const getSuggestion = <
-    T extends keyof SuggestionsMap = keyof SuggestionsMap
->(code: T) => suggestions[code]! as SuggestionsMap[T]
-
 export const suggestionFromGraphQLError = (
-    graphQLErrors: ReadonlyArray<GraphQLError>
-) => {
-    const type = fromGraphQLError(graphQLErrors)
+    apolloError: ApolloError
+): IKeepfyErrorTrusted => {
+    const [graphqlError = null] = apolloError.graphQLErrors as KeepfyGraphQLError[]
 
-    // Should we ignore other errors?
-    const [error] = graphQLErrors
-    const extracted = getSuggestion(type)
-    const properties = error.properties as { message?: string } | null
-    const message = properties && properties.message
-        ? properties.message // new format
-        : error.message // old format
-
-    // See error.ts extract from apollo fn
-    if(message) {
+    // only servers errors have the 'code'
+    if(graphqlError?.code) {
         return {
-            ...extracted,
-            message
+            type: graphqlError.code,
+            // if it is 'isTrusted', then it have a trusted message
+            message: graphqlError?.isTrusted
```

Comment:
precisa do '?.' aqui?

Reply:
talvez não, to hypado, to usando em todo lugar o '?.'